scripts/ci/run-jenkins.sh

```diff
@@ -105,7 +105,10 @@ if [[ ${CI_TAGS} == *'monolite'* ]]; then make get-monolite-latest; fi
 make_parallelism=-j4
 if [[ ${label} == 'debian-8-ppc64el' ]]; then make_parallelism=-j1; fi
 
-${TESTCMD} --label=make --timeout=${make_timeout} --fatal make ${make_parallelism} -w V=1
+if [[ ${CI_TAGS} != *'mac-sdk'* ]]; # Mac SDK builds Mono itself
+	then
```

Comment:
it probably makes more sense to move this check right at the top of the file so we can skip autogen.sh etc. as well and directly call into run-test-mac-sdk.sh:

```bash 
if [[ ${CI_TAGS} = *'mac-sdk'* ]]; # Mac SDK builds Mono itself
 	then
	${MONO_REPO_ROOT}/scripts/ci/run-test-mac-sdk.sh
	exit $?
fi
```

Reply:
I did it after autogen.sh on purpose as it takes care of inflating the submodules but I guess Jenkins can do that

Reply:
yes it does :)

Reply:
and bockbuild runs autogen.sh either doesn't it?